docs/opentelemetry/index.asciidoc

```diff
@@ -0,0 +1,60 @@
+the intention to achieve convergence of ECS and https://opentelemetry.io/docs/specs/semconv/[OTel Semantic Conventions (SemConv)].
+This donation should be seen as a directional decision for the evolution of both standards
+rather than a single event that merged both schemas into a single standard.
+
+While both schemes complement each other in large areas and thus offer mutual added value and great potential for convergence,
+it is important to understand that in some areas convergence is not achievable due to conceptual differences or consistency reasons. 
+For example, while ECS contains a few fields (e.g. https://www.elastic.co/guide/en/ecs/current/ecs-container.html#field-container-disk-read-bytes[container.disk.read.bytes])
+that are meant to be used in a metric-like way, metrics in OpenTelemetry are following a completely different data model (with metric name, type, dimensions, etc.).
+Also, in OTel semantic conventions there are some stable (or quasi-stable) attributes that have semantically equivalent, stable fields in ECS, however, with different field names.
+This kind of differences require explicit handling to achieve compatibility.
+
+The https://github.com/elastic/ecs/tree/main/schemas[ECS schema files] contain an explicit mapping between ECS fields and corresponding OTel semnatic convention attributes.
+This can be used to generate tooling for compatibility between ECS and semnatic conventions (e.g. alias fields in Elasticsearch).
+The relation between individual ECS fields and corresponding OTel semantic conventions attributes follows one of the following categories:
+
+[%header]
+|====
+| Category | Description
+
+| image:https://img.shields.io/badge/exact%20match-93c93e?style=flat[relation,title=match]
+| The name of the ECS field is identical to the SemConv attribute name and has (practically) the same semantics. 
+
+| image:https://img.shields.io/badge/equivalent-1ba9f5?style=flat[relation,title=equivalent]
+| The ECS field name is different but has the same semantics as the corresponding SemConv attribute.
+For this type of relation aliasing approaches (e.g. https://www.elastic.co/guide/en/elasticsearch/reference/current/field-alias.html[Elasticsearch field aliases]) can be used to achieve compatibility between ECS and OTel SemConv.
+
+| image:https://img.shields.io/badge/related-f2f4fb?style=flat[relation,title=related]
+| The ECS field name is different and has related - yet different - semantics as the corresponding SemConv attribute.
+An aliasing approach is not sufficient to resolve compatibility for this type of relation. 
+
+| image:https://img.shields.io/badge/metric-800080?style=flat[relation,title=metric]
+| For this ECS field there is a corresponding metric defined in OTel SemConv.
+
+| image:https://img.shields.io/badge/conflict-910000?style=flat[relation,title=conflict]
```

Comment:
should we also have a notation for fields that are never going to be donated to otel? such as `ecs.version`, probably `related.*` fields and maybe some others?

Reply:
Good point! I think that would make sense!

Reply:
added, a relation type `na`